client/landing/gutenboarding/onboarding-block/design-selector/index.tsx

```diff
@@ -93,6 +93,11 @@ const DesignSelector: React.FunctionComponent = () => {
 										className="design-selector__design-option"
 										onClick={ () => {
 											setSelectedDesign( design );
+
+											// Our design selector shows each template's default fonts, so the user expects to see those
+											// in the style preview. To match that expectation, we reset any previously user-selected fonts.
+											resetFonts();
```

Comment:
Alternatively, we could put the reset logic into `setSelectedDesign` (i.e. make the `selectedFonts` reducer listen to the `SET_SELECTED_DESIGN` action).

Reply:
I think your implementation is better. We might also use this action later.